### PR TITLE
Project Update to Xcode 9.3

### PR DESCRIPTION
**Pull request by ghost**
* updates all Xcode project files to current version 9.3
* adds `.travis.yml` to the Xcode project - for convenience
* removes unnecessary `brew install carthage` config from `.travis.yml`
* makes some changes in class `MemoryQueueSpec` to avoid warnings, local variable `queue` is now a constant

### CODE REVIEW

**Comment by ghost** [comment]
@brototyp Please review.

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,11 @@
 language: objective-c
+osx_image: xcode9.2
```
Comment (brototyp) [inline .travis.yml:2]: We could select xcode9.3 here as well, right?

File: MatomoTrackerTests/MemoryQueueSpec.swift [hunk 2]
```diff
@@ -93,7 +93,7 @@ class MemoryQueueSpec: QuickSpec {
         }
         describe("remove") {
             it("should remove events that are enqueued") {
-                var queue = MemoryQueueFixture.withTwoItems()
+                let queue = MemoryQueueFixture.withTwoItems()
```
Comment (brototyp) [inline MatomoTrackerTests/MemoryQueueSpec.swift:96]: 👍

**Review by brototyp — APPROVED** [review]
I am happy with this. Feel free to update the osx_image to `xcode9.3` if you feel this to be a worthwhile thing.

File: .travis.yml (hunk 1 above)
Comment (ghost) [reply .travis.yml:2]: Yes, but then Swift 4.1 gets selected, which results in deprecation warnings.

Comment (brototyp) [reply .travis.yml:2]: Alright. Good point. Let's to that when we migrate to 4.1 then. Thanks.

**Pull request merged**